tools/foxglove/cereal_to_mcap.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
+
+import cereal
```

Comment:
imports should be sorted by std lib and third party, newline, then openpiiot imports